PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -43,17 +43,22 @@ class ArchivedItemsListViewModel: ItemsListViewModel {
     private var selectedFilters: Set<ItemsListFilter> = .init()
     private let availableFilters: [ItemsListFilter] = ItemsListFilter.allCases
 
-    init(source: Source, tracker: Tracker) {
+    init(
+        source: Source,
+        tracker: Tracker,
+        networkMonitor: NetworkPathMonitor = NWPathMonitor()
+    ) {
         self.source = source
         self.tracker = tracker
+        self.networkMonitor = networkMonitor
 
         networkMonitor.start(queue: .global())
 //        self.main.$selectedMyListReadableViewModel.sink { _ in
 //            // TODO: Handle deselection here
 //        }.store(in: &subscriptions)
     }
 
-    func fetch() async throws {
+    private func doFetch() async throws {
```

Comment:
Tiny tidbit for cleanup: I'm partial to moving private functions either together, or in an extension (I typically prefer an extension), just to make the public API "flow" a bit more. Thoughts?

Reply:
Discussed offline: we extracted groups of related functions into their own extensions